apps/admin_api/test/admin_api/v1/controllers/provider_auth/key_controller_test.exs

```diff
@@ -119,6 +124,36 @@ defmodule AdminAPI.V1.ProviderAuth.KeyControllerTest do
     end
   end
 
+  describe "/access_key.enable_or_disable" do
```

Comment:
Can you add a test that disabling it twice doesn't make it enabled?

Just in case someone turns it into a toggle in the future.